packages/sources/dxfeed/src/endpoint/price.ts

```diff
@@ -19,9 +25,12 @@ export const execute: ExecuteWithConfig<Config> = async (request, _, config) =>
   if (validator.error) throw validator.error
 
   const jobRunID = validator.validated.id
-  const symbol = (validator.overrideSymbol(AdapterName) as string).toUpperCase()
-  const events = quoteEventSymbols[symbol] ? 'Quote' : 'Trade'
+  const base = validator.validated.data.base
+  const symbol = Array.isArray(base)
+    ? base.map((symbol) => symbol.toUpperCase()).join(',')
```

Comment:
Just a style thing, can we remove the parentheses around `symbol`?  So `base.map(symbol => symbol.toUpperCase()).join(',')`.

Reply:
done! :)